data/src/normalize.py

```diff
@@ -0,0 +1,39 @@
+import argparse
+
+from tempfile import TemporaryFile
+from unicodedata import normalize
+
+'''
```

Comment:
1. This has to go above any imports for Python to understand that it's documentatino.
2. Don't include the program name: that's self-evident.
3. The first sentence should be a brief <80 explanation, then you go into normal prose. E.g.:

```
"""In-place Unicode normalization.

This script takes a file and normalizes it in-place...
"""
```

4. Use " not '. " is a better string-quote character.